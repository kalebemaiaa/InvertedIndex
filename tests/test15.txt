QueryOr(QueryAndS(QueryOr(QueryWord("chefe"), QueryAnd(QueryWord("coordenador"), QueryOr(QueryWord("embaixador"), QueryWord("secretário")))), QueryWord("diplomata")), QueryAndS(QueryOr(QueryWord("governador"), QueryWord("prefeito")), QueryAnd(QueryWord("assessor"), QueryOr(QueryWord("ministro"), QueryWord("presidente")))))
4976.raw
3525.raw
2562.raw
8756.raw
5763.raw
3688.raw
1815.raw
5792.raw
1128.raw
5234.raw
2985.raw
11252.raw
303.raw
4309.raw
5948.raw
1202.raw
3883.raw
2017.raw
222.raw
11431.raw
11260.raw
227.raw
4745.raw
2030.raw
2110.raw
3142.raw
431.raw
5165.raw
2997.raw
11914.raw
4959.raw
3128.raw
2444.raw
3151.raw
10983.raw
5232.raw
630.raw
811.raw
2452.raw
4401.raw
3062.raw
1875.raw
4585.raw
572.raw
910.raw
11093.raw
11223.raw
4821.raw
4570.raw
285.raw
2531.raw
3641.raw
5791.raw
5404.raw
4882.raw
11863.raw
947.raw
10900.raw
11734.raw
6142.raw
4348.raw
4872.raw
5641.raw
3086.raw
10928.raw
11606.raw
11166.raw
1587.raw
5619.raw
4629.raw
2554.raw
2522.raw
2678.raw
922.raw
4043.raw
1091.raw
4329.raw
11814.raw
488.raw
3172.raw
917.raw
2589.raw
2597.raw
5504.raw
11237.raw
4095.raw
11696.raw
3765.raw
256.raw
279.raw
217.raw
4661.raw
1906.raw
1689.raw
11511.raw
10861.raw
5298.raw
1215.raw
3537.raw
3677.raw
2708.raw
826.raw
4002.raw
11543.raw
178.raw
5593.raw
1769.raw
5063.raw
3145.raw
1428.raw
4380.raw
1958.raw
4990.raw
850.raw
3478.raw
132.raw
1158.raw
11680.raw
6173.raw
11216.raw
3189.raw
11960.raw
724.raw
5132.raw
3137.raw
1132.raw
5860.raw
4900.raw
6119.raw
2208.raw
4797.raw
2414.raw
5103.raw
11149.raw
3917.raw
3859.raw
1993.raw
670.raw
2063.raw
6162.raw
11845.raw
5121.raw
878.raw
5035.raw
4572.raw
11971.raw
5736.raw
2518.raw
3608.raw
15.raw
11570.raw
1326.raw
5724.raw
11705.raw
1592.raw
5711.raw
350.raw
10864.raw
2513.raw
3947.raw
4083.raw
2574.raw
3301.raw
581.raw
393.raw
11633.raw
4381.raw
6404.raw
11308.raw
11155.raw
2131.raw
2790.raw
516.raw
5173.raw
3386.raw
3735.raw
2884.raw
5549.raw
2874.raw
5955.raw
4101.raw
11214.raw
116.raw
219.raw
4607.raw
11217.raw
3173.raw
3291.raw
4888.raw
3674.raw
2393.raw
11621.raw
6359.raw
5998.raw
5633.raw
3927.raw
4588.raw
420.raw
12030.raw
5539.raw
