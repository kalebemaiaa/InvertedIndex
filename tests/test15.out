826
910
1326
3765
4043
5121
5593
5792
11166
11570
11680
